documentation/releaseNotes/releaseNotes.md

```diff
@@ -2,5 +2,6 @@ Today we are releasing the next preview of the `dotnet monitor` tool. This relea
 
 - Load a profiler via a collection rule action (#781)
 - Get or Set an environment via a collection rule action (#1176)
+- Add `MachineJson` option to `--output` parameter for `generatekey` command. This is useful for automating credential provisioning. (#1428)
```

Comment:
I didn't think this change was part of our 6.1 release?

Reply:
Ahh, I thought this was going to become the `7.0` release notes. I should make a separate `7.0`/`6.2` release notes